src/models/Article.php

```diff
@@ -174,8 +195,9 @@ public function ngRestAttributeGroups()
     public function ngRestScopes()
     {
         return [
-            [['list'], ['cat_id', 'title', 'timestamp_create', 'image_id']],
-            [['create', 'update'], ['cat_id', 'title', 'teaser_text', 'text', 'timestamp_create', 'timestamp_display_from', 'is_display_limit', 'timestamp_display_until', 'image_id', 'image_list', 'file_list']],
+            [['list'], ['id', 'cat_id', 'slug', 'title', 'timestamp_create', 'image_id']],
+            [['create'], ['slug', 'cat_id', 'title', 'teaser_text', 'text', 'timestamp_create', 'timestamp_display_from', 'is_display_limit', 'timestamp_display_until', 'image_id', 'image_list', 'file_list']],
```

Comment:
as create an update are the same now we could write [['create', 'update'], .....] instead of copy the row.